src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Extensions/ConfigurationClientExtensions.cs

```diff
@@ -3,12 +3,11 @@
 //
 using Azure;
 using Azure.Data.AppConfiguration;
-using Microsoft.Extensions.Configuration.AzureAppConfiguration.FeatureManagement;
+using Microsoft.Extensions.Configuration.AzureAppConfiguration.Models;
 using System;
 using System.Collections.Generic;
 using System.Linq;
```

Comment:
This seems to be unused.

Reply:
Used for `KeyValueIdentifier` at the moment

Reply:
I mean the line 9 `System.Linq`

Reply:
oh sorry haha yes removed